Alisha/data/lab1.ipynb

```diff
@@ -0,0 +1,354 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    #Question No. 3 **Scatter Plot with Random Data**  
</pre><p>Inconsistent comment. Either only add comment or add this in a separate cell (markdown) above the code cell</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/suparthghimire/data-analysis-and-visualization-bca-2025/pull/6/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>